src/Domain/Insights/ForbiddenSecurityIssues.php

```diff
@@ -80,7 +80,7 @@ private function getResult(): void
             array_map(static fn (array $detail): string => $detail['version'], $packages['packages'])
         );
 
-        if ($packagesToCheck === false) {
+        if ($packagesToCheck == false) {
```

Comment:
any reason to remove the strict comparison ?

Reply:
Yes, PHPstan was complaining about this:

`Strict comparison using === between array<string, string> and false will always evaluate to false.`

Reply:
:thinking: that means `$packagesToCheck` is always an array so ? Maybe this check is not necessary at all :sweat_smile:

Reply:
Yes, you're right! I think we can skip this check. Want me to remove it?

Reply:
I think that's something that differ between php7 and php8. 
Seems the array_combine function return false in case of the two array passed are not the same size in php7, and an error is thrown in php8. (see https://3v4l.org/8LD17)

But maybe phpstan can now check the size of arrays passed to the function, so maybe that don't matter.

You can try to remove it :+1: